README.md

```diff
@@ -1,23 +1,279 @@
+        .open_or_create()
+        .expect("failed to create/open event");
+
+    let notifier = event
+        .notifier()
+        .create()
+        .expect("failed to create notifier");
+
+    let mut counter: u64 = 0;
+    while !SignalHandler::termination_requested() {
+        counter += 1;
+        notifier
+            .notify_with_custom_trigger_id(EventId::new(counter))
+            .expect("failed to trigger event");
+
+        println!("Trigger event with id {} ...", counter);
+        std::thread::sleep(std::time::Duration::from_secs(1));
+    }
+}
+```
+
+**listener.rs**
+
+```rust
+use elkodon::prelude::*;
+use elkodon_bb_posix::signal::SignalHandler;
+
+fn main() {
+    let event_name = ServiceName::new(b"MyEventName").unwrap();
+
+    let event = zero_copy::Service::new(&event_name)
+        .event()
+        .open_or_create()
+        .expect("failed to create/open event");
+
+    let mut listener = event
+        .listener()
+        .create()
+        .expect("failed to create listener");
+
+    while !SignalHandler::termination_requested() {
+        for event_id in listener
+            .timed_wait(std::time::Duration::from_secs(1))
+            .expect("failed to wait on listener")
+        {
+            println!("event was triggered with id: {:?}", event_id);
+        }
+    }
+}
+```
+
+This example is a simplified version of the
+[event example](examples/examples/event/). You can
+execute it by opening two terminals and calling:
+
+**Terminal 1:**
+
+```sh
+cargo run --example event_notifier
+```
+
+**Terminal 2:**
+
+```sh
+cargo run --example event_listener
+```
+
 ## Supported Platforms
 
-### Target Platforms
+The support levels can be adjusted when required.
+
+| Operating System | State        | Current Support Level | Target Support Level |
+|------------------|:-------------|:---------------------:|---------------------:|
+| Android          | planned      | tier 3                | tier 1               |
+| FreeBSD          | done         | tier 1                | tier 1               |
+| FreeRTOS         | planned      | tier 3                | tier 2               |
+| iOS              | planned      | tier 3                | tier 2               |
+| Linux (x86_64)   | done         | tier 1                | tier 1               |
+| Linux (aarch64)  | done         | tier 1                | tier 1               |
+| Linux (32-bit)   | in-progress  | tier 3                | tier 1               |
+| Mac OS           | in-progress  | tier 3                | tier 2               |
+| QNX              | planned      | tier 3                | tier 1               |
+| WatchOS          | planned      | tier 3                | tier 2               |
+| Windows          | done         | tier 2                | tier 2               |
+
+- **tier 1** - All safety and security features are working.
+- **tier 2** - Works with a restricted security and safety feature set.
+- **tier 3** - Not yet working but planned.
```

Comment:
What do you think of this?

```suggestion
| Operating System | State        | Current Support Level | Target Support Level |
|------------------|:-------------|:---------------------:|---------------------:|
| Android          | planned      | ---                   | tier 1               |
| FreeBSD          | done         | tier 1                | tier 1               |
| FreeRTOS         | planned      | ---                   | tier 2               |
| iOS              | planned      | ---                   | tier 2               |
| Linux (x86_64)   | done         | tier 1                | tier 1               |
| Linux (aarch64)  | done         | tier 1                | tier 1               |
| Linux (32-bit)   | in-progress  | tier 3                | tier 1               |
| Mac OS           | in-progress  | tier 3                | tier 2               |
| QNX              | planned      | ---                   | tier 1               |
| WatchOS          | planned      | ---                   | tier 2               |
| Windows          | done         | tier 2                | tier 2               |

- **tier 1** - All safety and security features are working.
- **tier 2** - Works with a restricted security and safety feature set.
- **tier 3** - Work in progress. Might compile and run or not.
```

I'm even a bit hesitant to mark Linux as done and tier 1 at the moment. Not all communication features are implemented and I don't know the current state of all the implemented safety features.